package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "never-consent",
-  "version": "1.15.3",
```

Comment:
same question

Reply:
Sorry, I think the build script erased those.